src/applications/disability-benefits/526EZ/reference-helpers.js

```diff
@@ -1,8 +1,841 @@
 import Raven from 'raven-js';
 
 const diagnosticCodeMap = {
-  '1234': 'PTSD', // eslint-disable-line
-  '5678': 'Intervertebral Disc Syndrome' // eslint-disable-line
+  10: 'Abnormal Heart',
```

Comment:
I probably missed capitalizations, so a careful review of this chunk would be appreciated.

Also, @peggygannon I know you edited this list once before, but I guess they updated it so we couldn't use that edited version. I took the version we were given and capitalized all the words except "of," "and," and the like. If you could give this a pass as well and point out any changes you have in mind, that would be awesome.

Reply:
@cvalarida thanks, Chris. I took a look and made minor copy edits in the file. (capping or lowercasing pronouns, etc)

Reply:
Beautiful! Thanks, @peggygannon! :raised_hands: